src/hooks/useScrollTabIntoView.ts

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect, useRef } from "react";
+
+export const useScrollTabIntoView = (index: number) => {
```

Comment:
Can you add a comment block that this is meant to only be used (as of now it seems) in the `Tabs` component. Also that this is an internal hook that is not meant to be used by devs outside of the DS or `Tabs` component. There are other similar non-consuming app hooks like `useDSHeading` and `useStateWithDependencies`.